### PR TITLE
Select Pipeline name (if applicable) when navigating to Create Run page

### PR DESCRIPTION
**Pull request by diegolovison**

### CODE REVIEW

**Review by mwaykole — APPROVED** [review]

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource [hunk 1]
```diff
@@ -40,14 +40,18 @@ Fill In Run Creation Form    # robocop: disable
     [Documentation]    Compiles the form to create a pipeline run.
     ...                It works when you start server creation from either
     ...                DS Project details page or DS Pipelines page.
-    [Arguments]    ${name}    ${pipeline_name}    ${from_actions}=${TRUE}    ${run_type}=Immediate
-    ...            ${trigger_type}=Periodic    ${start_date}=${NONE}    ${start_time}=${NONE}
+    [Arguments]    ${name}    ${pipeline_name}    ${from_actions_menu}    ${from_actions}=${TRUE}
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource:43]: what's the difference btw these 2 arguments?

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource [hunk 2]
```diff
@@ -81,7 +81,7 @@ Create Pipeline Run    # robocop: disable
     Fill In Run Creation Form    name=${name}    pipeline_name=${pipeline_name}
     ...    run_type=${run_type}    trigger_type=Periodic    start_date=${start_date}
     ...    start_time=${start_time}    end_date=${end_date}    end_time=${end_time}
-    ...    cron_expr=${cron_expr}    &{model_param}
+    ...    cron_expr=${cron_expr}    &{model_param}    from_actions_menu=${from_actions_menu}
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource:84]: the argument is not used in the code of this keyword. could you remove it?

Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource:84]: the argument is not used in the code of this keyword. could you remove it?

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource [hunk 3]
```diff
@@ -40,14 +40,18 @@ Fill In Run Creation Form    # robocop: disable
...
+    ...            ${run_type}=Immediate    ${trigger_type}=Periodic    ${start_date}=${NONE}    ${start_time}=${NONE}
     ...            ${end_date}=${NONE}    ${end_time}=${NONE}    ${cron_expr}=${NONE}&{model_param}
     Input Text    ${PIPELINE_NAME_INPUT_XP}    ${name}
     Input Text    ${PIPELINE_DESC_INPUT_XP}    ${name}
-    Run Keyword And Continue On Failure
-    ...    Element Should Be Disabled    ${GENERIC_CREATE_BTN_XP}
-    Select Pipeline For The Run    pipeline_name=${pipeline_name}
+    IF    ${from_actions_menu} == ${FALSE}
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource:48]: does the issue happens only when creating the run not from the actions menu?

**Review by bdattoma — CHANGES_REQUESTED** [review]

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource [hunk 4]
```diff
@@ -40,14 +40,18 @@ Fill In Run Creation Form    # robocop: disable
...
+        Wait Until Page Contains Element    xpath://button[span[text()='${pipeline_name}']]
+    ELSE
+        Run Keyword And Continue On Failure
+        ...    Element Should Be Disabled    ${GENERIC_CREATE_BTN_XP}
+        Select Pipeline For The Run    pipeline_name=${pipeline_name}
```
Comment (bdattoma) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataSciencePipelines.resource:53]: hm probably, in order to warn when the issue happens, we could use a different flow like:

```
IF ${GENERIC_CREATE_BTN_XP} is enabled when it should not
THEN raise a WARN message and Wait until the dropdown gets the value
```

**Review by bdattoma — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]

**Comment by diegolovison** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/1853/robot/report/files/test_report.html

**Pull request merged**